<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Remesh_planar_patches_dialog</class>
 <widget class="QDialog" name="Remesh_planar_patches_dialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>262</width>
    <height>285</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Remesh planar patches parameters</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Parameters</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="cos_label">
          <property name="text">
           <string>Cosinus Threshold</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="DoubleEdit" name="cos_dspinbox">
          <property name="inputMethodHints">
           <set>Qt::ImhNone</set>
          </property>
          <property name="inputMask">
           <string/>
          </property>
          <property name="text">
           <string>1</string>
          </property>
          <property name="placeholderText">
           <string>0.00</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="dist_label">
          <property name="text">
           <string>Frechet Distance</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="DoubleEdit" name="dist_dspinbox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="inputMethodHints">
           <set>Qt::ImhNone</set>
          </property>
          <property name="inputMask">
           <string/>
          </property>
          <property name="text">
           <string>1</string>
          </property>
          <property name="placeholderText">
           <string>0.00</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="notriangulation_checkbox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Do not triangulate patches</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="use_region_growing_checkbox">
        <property name="text">
         <string>Use region growing of planes</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="postprocess_regions_checkbox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Post-process regions</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="create_new_item_checkbox">
        <property name="text">
         <string>Create new item</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DoubleEdit</class>
   <extends>QLineEdit</extends>
   <header>CGAL_double_edit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cos_dspinbox</tabstop>
  <tabstop>notriangulation_checkbox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Remesh_planar_patches_dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>397</x>
     <y>333</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Remesh_planar_patches_dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>397</x>
     <y>333</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>use_region_growing_checkbox</sender>
   <signal>toggled(bool)</signal>
   <receiver>postprocess_regions_checkbox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>143</x>
     <y>214</y>
    </hint>
    <hint type="destinationlabel">
     <x>143</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>use_region_growing_checkbox</sender>
   <signal>toggled(bool)</signal>
   <receiver>dist_dspinbox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>143</x>
     <y>214</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
